### PR TITLE
[9/x] only show summary stats if user has tokens

### PR DESCRIPTION
**Pull request by jmilldotdev**
## What does this PR do and why?

Move summary stats section into OwnedTokensSection and only display if the user has tokens.

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by jmilldotdev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1654** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1654" target="_blank">Graphite</a> 
    * **PR #1655** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1655" target="_blank">Graphite</a> 
      * **PR #1656** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1656" target="_blank">Graphite</a> 
        * **PR #1657** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1657" target="_blank">Graphite</a> 
          * **PR #1658** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1658" target="_blank">Graphite</a> 
            * **PR #1659** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1659" target="_blank">Graphite</a> 
              * **PR #1660** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1660" target="_blank">Graphite</a> 
                * **PR #1661** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1661" target="_blank">Graphite</a> 
                  * **PR #1662** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1662" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1662?utm_source=stack-comment).

File: src/components/veNft/VeNftOwnedTokensSection.tsx [hunk 1]
```diff
@@ -27,26 +28,34 @@ export default function OwnedNFTSection({
   const hasOverflow = Boolean(primaryTerminalCurrentOverflow?.gt(0))
 
   return (
-    <div style={{ ...shadowCard(theme), padding: 25, marginBottom: 10 }}>
+    <>
```
Comment (tomquirk) [inline src/components/veNft/VeNftOwnedTokensSection.tsx:31]: **question**: does it still work if we remove the wrapping fragment?

**Review by tomquirk — DISMISSED** [review]
See inline comments

File: src/components/veNft/VeNftOwnedTokensSection.tsx (hunk 1 above)
Comment (jmilldotdev) [reply src/components/veNft/VeNftOwnedTokensSection.tsx]: it doesn't, i could use a div

**Review by tomquirk — APPROVED** [review]

**Pull request merged**